x-pack/plugin/ilm/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/ilm/11_basic_slm.yml

```diff
@@ -24,6 +24,20 @@ setup:
           settings:
             location: "my-snaps"
 
+  - do:
+      catch: bad_request
```

Comment:
We should catch the actual exception here with something like:

```
catch: /invalid schedule \[0 * * * * ?\]: too frequent/
```

(with whatever the final message ends up being)

Reply:
✅, 2798b78608197e770d3bbf02e2639076da30de1c.